addons/mail/static/src/rtc/call_participant_card.scss

```diff
@@ -34,11 +35,16 @@
     85% { border: solid $gray-700 }
 }
 
+.o-mail-CallParticipantCard-iconBlackBg {
+    background-color: rgba(0, 0, 0, 0.75);
```

Comment:
This could be `.bg-black-75` ?

Reply:
No, `bg-black-75` becomes white in dark mode